server/package.json

```diff
@@ -87,14 +87,19 @@
     "joi": "^14.3.1",
     "jsonwebtoken": "^8.4.0",
     "lodash.clonedeep": "^4.5.0",
+    "mock-http-server": "^1.4.2",
```

Comment:
Where is it used?

Reply:
mock-http-server is used in withings test, I put it in devDependencies